src/SIUnits.jl

```diff
@@ -7,15 +7,28 @@ module SIUnits
     import Base: ==, +, -, *, /, .+, .-, .*, ./, //, ^
     import Base: promote_rule, promote_type, convert, show, mod
 
-    immutable SIQuantity{T<:Number,m,kg,s,A,K,mol,cd,rad,sr} <: Number
+    typealias UnitTuple NTuple{9,Int64}
```

Comment:
Is `Int64` really the correct thing here? Will this still be able to represent units like `s^(1/2)`?